### PR TITLE
Development documentation

### PR DESCRIPTION
**Pull request by tokejepsen**
Getting started for developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,13 @@ $ net use a: \\192.168.99.100\avalon
...
 ### Development
 
-Use the overrides to mount your local development directory in place of the deployed version.
+For development [Git](https://git-scm.com/) is required.
 
 ```bash
-$ docker-compose -f docker-compose.yml -f docker-compose-dev.yml up
+$ git clone https://github.com/getavalon/docker.git --recursive
+$ cd docker
+$ docker build . -t avalon/latest
```
Comment (mottosso) [inline README.md]: Think you mean `getavalon/docker:latest`, where `getavalon` is our Docker Hub org and `latest` is the version. Though for development, you can omit `latest` altogether, it's mostly relevant for users.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 # docker
 
-Avalon on [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/), ideal for beginners and studios alike.
```
Comment (mottosso) [inline README.md]: We do need to be explicit about Toolbox, as opposed to "For Windows", as Toolbox is more hidden, not as obvious to install. And this only works with Toolbox.

File: README.md [hunk 3]
```diff
@@ -28,8 +28,13 @@ $ net use a: \\192.168.99.100\avalon
 
 ### Development
 
-Use the overrides to mount your local development directory in place of the deployed version.
+For development [Git](https://git-scm.com/) is required.
```
Comment (mottosso) [inline README.md]: I don't think it needs to be said on GitHub that you need Git to contribute. :)

File: README.md (hunk 2 above)
Comment (tokejepsen) [reply README.md]: Yeah, but we are linking to the Windows installation. Replaced that with the Windows and Mac installation instructions.

**Comment by tokejepsen** [comment]
Good to merge?

**Pull request merged**